development/test-log-collector/cmd/app/mainerr.go

```diff
@@ -0,0 +1,259 @@
+package app
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"io"
+	"os"
+	"time"
+
+	logf "github.com/sirupsen/logrus"
+	slackGo "github.com/slack-go/slack"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+
+	pkgConfig "github.com/kyma-project/test-infra/development/test-log-collector/pkg/config"
+	"github.com/kyma-project/test-infra/development/test-log-collector/pkg/slack"
+
+	pkgSlack "github.com/kyma-project/test-infra/development/test-log-collector/pkg/slack"
+
+	"github.com/pkg/errors"
+	"github.com/vrischmann/envconfig"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/client-go/kubernetes"
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+	restclient "k8s.io/client-go/rest"
+
+	"github.com/kyma-project/test-infra/development/test-log-collector/pkg/hyperscaler"
+
+	"github.com/kyma-project/test-infra/development/test-log-collector/pkg/resources/clustertestsuite"
+	octopusTypes "github.com/kyma-project/test-infra/development/test-log-collector/pkg/resources/clustertestsuite/types"
+)
+
+type config struct {
+	SlackToken     string
```

Comment:
I understand that the job is only supposed to work for one workspace. What if someone wants to have data sent to another workspace than our main? :)

Reply:
But this not a critical change, it can be given when someone needs this possibility

Reply:
Given that we have one main workspace where we work it shouldn't be a problem, and if someone _really_ needs it, then s/he can add correct logic for it as a contribution